scripts/buidl.sh

```diff
@@ -153,19 +159,45 @@ print_output() {
     echo "injected data: '$(< "$default_nix" tr '\n' ' ' | tr -s ' ' | sed 's/ $//')'"
   fi
 
-  # Print the real paths of the symlinks
-  if [ $dry_run = false ]; then
-    for symlink in "$output_path"/*; do
-      real_path=$(readlink -f "$symlink")
-      if [ "$verbose" = true ]; then 
-        echo created symlink: \'"$symlink > $real_path"\'
+  # Print the paths of the results
+  if [ "$dry_run" = false ]; then
+    for file_path in "$output_path"/*; do
+      if [ -h "$file_path" ]; then
+        real_path=$(readlink -f "$file_path")
+        if [ "$verbose" = true ]; then 
+          echo "created symlink: '$file_path > $real_path'"
+        else
+          echo "$real_path"
+        fi
       else
-        echo "$real_path"
+        if [ "$verbose" = true ]; then
+          echo "created file: '$file_path'"
+        else
+          echo "$file_path"
+        fi
       fi
     done
   fi
 }
 
+get_result() {
+    local hostname="$1"
+    local output_path="$2"
+    local -a nix_flags=("${@:3}")
+
+    # Get the path to the Nix store
+    kexec_tree=$(nix eval --raw .#nixosConfigurations."$hostname".config.system.build.kexecTree "${nix_flags[@]}")
+
+    # Copy the files to the output path
+    mkdir -p "$output_path"
+
+    for symlink in "$kexec_tree"/*; do
+      real_path=$(readlink -f "$symlink")
+      new_real_path="$output_path/$(basename "$symlink")"
+      cp -f "$symlink" "$new_real_path"
+    done
+}
+
 create_webui_files() {
```

Comment:
Can we have this as a separate script? We need to run this every time the web server is started.

Reply:
Sure, doing this tomorrow.